<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QWidget" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_toMain">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Limelight</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #16082f; border: none; border-radius: 15px;
color: white;
}
                         QPushButton:hover {
                           background-color: #18174f; /* Цвет фона при наведении */
                         }
                         QPushButton:pressed {
                         background-color: #18174ffe; /* Цвет фона при нажатии */
color: grey;
                        }
</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>801</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_totalS">
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Общая сумма кредита (руб.)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_totalS">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label__overpayment">
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Переплата по кредиту</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_overpayment_res">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
qproperty-alignment: 'AlignCenter';
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label__total_payout">
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Общая выплата</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_total_payout_res">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
qproperty-alignment: 'AlignCenter';
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_percent">
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Процентная ставка (%)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_percent">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_term">
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Срок кредита</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_term">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_temp">
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Месяцев</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Лет</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_payment">
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ежемесячный платеж</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_payment_res">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
qproperty-alignment: 'AlignCenter';
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_calculate">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Limelight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #16082f; border: none; border-radius: 15px;
color: white;
}
                         QPushButton:hover {
                           background-color: #18174f; /* Цвет фона при наведении */
                         }
                         QPushButton:pressed {
                         background-color: #18174ffe; /* Цвет фона при нажатии */
color: grey;
                        }
</string>
      </property>
      <property name="text">
       <string>Посчитать</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label__credittype">
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Тип кредита</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_credittype">
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Аннуитентный</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Дифференцированный</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="2" rowspan="3">
     <widget class="QListWidget" name="listWidget_diff_payout">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>1000</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
